### DISCUSSION TITLE
How to get admin email and password?

### DISCUSSION THREAD

**Discussion by CodeWithCJ** [alexjustesen/speedtest-tracker#1854] [Q&A] [answered]
I deleted volume folder multiple times in my ubuntu.  But still it asks for admin email and passoword. not allowing me to signup.

**Comment by alexjustesen** [comment]
You can use the `app:` commands to reset your password https://docs.speedtest-tracker.dev/other/commands

**Reply by svenvg93** [reply, marked as answer]
Or use the default username and password listed here: https://docs.speedtest-tracker.dev/security/authentication ;)
